utils/statediff/Cargo.toml

```
@@ -1,5 +1,7 @@
 [package]
 name = "forest_statediff"
+description = "statediff binary sub-commands available"
```

The old description seemed odd; perhaps we can make it more like a description.
```suggestion
description = "Utility tool for debugging Filecoin state differences."
```

---

Fixed